script/make_release.py

```diff
@@ -29,35 +29,29 @@ def confirm(message, printMessage):
     quit()
 
 
-def getLatestChangelogEntry(path):
-  release_post_content = ""
+def getLatestChangelogEntries(path):
   with open(path, "r", encoding="utf-8") as fp:
     line_number = 0
-    section = None
+    new_changelog_section = ""
+
     for line in fp:
-      if line_number > 20:
+      if line_number > 10:
+
+        # break on headline for already released log entries
         if "## [" in line:
```

Comment:
While being on it: we should likely make this a bit safer (even though it's unlikely that `## [` is anywhere in the middle of the line) by checking that it matches the start of the line.

```suggestion
        if line.startswith("## [")
```